scripts/aem.js

```diff
@@ -537,7 +537,7 @@ function buildBlock(blockName, content) {
       vals.forEach((val) => {
         if (val) {
           if (typeof val === 'string') {
-            colEl.innerHTML += val;
+            colEl.textContent += val;
```

Comment:
Note that the `aem.js` source is [here](https://github.com/adobe/aem-lib) and the build result gets copied to the boilerplate.